formula-rxjava3/src/main/java/com/instacart/formula/rxjava3/RxDisposableAction.kt

```diff
@@ -0,0 +1,66 @@
+package com.instacart.formula.rxjava3
+
+import com.instacart.formula.Cancelable
+import com.instacart.formula.DisposableAction
+import io.reactivex.rxjava3.core.Observable
+
+/**
+ * Adapter to convert RxJava3 observables to [DisposableAction].
+ */
+interface RxDisposableAction<Message> : DisposableAction<Message> {
```

Comment:
I wonder if this should be replaced with extension functions on `DisposableAction` such that you would do
```kotlin
import com.instacart.formula.DisposableAction
import com.instacart.formula.rxjava3.fromObservable

DisposableAction.fromObservable { Observable.just(1, 2, 3) }.onEvent { event ->
    transition()
}
```

versus
```kotlin
import com.instacart.formula.rxjava3.RxDisposableAction

RxDisposableAction.fromObservable { Observable.just(1, 2, 3) }.onEvent { event ->
    transition()
}
```